docs/docs/resources/pipeline-components/kafka-connect.yaml

```diff
@@ -0,0 +1,44 @@
+- type: kafka-connect # required
+  name: kafka-connect # required
+  namespace: namespace # required, clean up jobs for the connector will run here
+  # `app` contains application-specific settings, hence it does not have a rigid
+  # structure. The fields below are just an example. Extensive documentation on
+  # connectors: https://kafka.apache.org/documentation/#connectconfigs
+  app:
+    tasks.max: 1
+  # Topic(s) from which the component will read input
+  from:
+    topics: # required
+      ${pipeline_name}-input-topic:
+        type: input # required
+        # role: topic-role # only used if type is `extra`
+  # Topic(s) into which the component will write output
+  to:
+    topics: # required
+      ${pipeline_name}-output-topic:
+        type: error # required
+        keySchema: key-schema # must implement SchemaProvider to use
+        valueSchema: value-schema
+        partitions_count: 1
+        replication_factor: 1
+        configs: # https://kafka.apache.org/documentation/#topicconfigs
+          cleanup.policy: compact
+    models: # SchemaProvider is initiated with the values given here
+      model: model
+  # Pipeline prefix that will prefix every component name. If you wish to not 
+  # have any prefix you can specify an empty string.
+  prefix: ${pipeline_name}-
+  # Helm repository configuration for resetter
+  repoConfig:
+    repositoryName: my-repo # required
+    url: https://bakdata.github.io/kafka-connect-resetter/ # required
+    repoAuthFlags: 
+      username: user
+      password: pass
+      ca_file: /home/user/path/to/ca-file
+      insecure_skip_tls_verify: false
+  version: "1.0.4" # Helm chart version
+  # Overriding Kafka Connect Resetter Helm values. E.g. to override the
+  # Image Tag etc.
+  resetterValues:
+    imageTag: 1.2.3
```

Comment:
Minor: `version` is with double quotes, and `imageTag` is without. Maybe we should be consistent on that

Reply:
Does it matter to KPOps?

Reply:
I quated all semantic versions that I found.